portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase

```diff
@@ -74,4 +141,140 @@ definition {
 			vocabularyName = "Vocabulary Name");
 	}
 
+	@description = "This is a use case for LPS-107730. Can input 254 characters to name field, key field and value field of category."
+	@priority = "4"
+	test Input254CharactersIntoNameFieldKeyFieldAndValueField {
+		var inputText = "The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 25";
```

Comment:
It has 255 characters, is it right?

Reply:
The value of inputText has 254 characters.